src/core/thread/mle.cpp

```diff
@@ -145,6 +148,34 @@ Mle::Mle(Instance &aInstance)
     SetMeshLocalPrefix(AsCoreType(&kMeshLocalPrefixInit));
 }
 
+void Mle::HandleResetTimer()
```

Comment:
Consider if we want to do this whole thing differently with an explicit attach state in MLE rather than kind of going around with a timer.